pybio/spec/schema.py

```diff
@@ -0,0 +1,253 @@
+from dataclasses import asdict
+from typing import Any, Dict, Union
+
+from marshmallow import Schema as _Schema, pprint, ValidationError, post_load, validates_schema, validates
+
+from pybio.spec import pybio_types, fields
+from pybio.spec.pybio_types import MagicTensorsValue, MagicShapeValue
+
+
+class Schema(_Schema):
```

Comment:
Is it maybe a bit confusing that we call this `Schema` and inherit from `marshmallow.Schema`?
We could call it `PybioSchema` and then also avoid the `from marshmallow import Schema as _Schema`.